### PR TITLE
Outpost45update

### PR DESCRIPTION
**Pull request by Yobrocharlie**
## About The Pull Request

Outpost45 is as of now, over eight years old, and after so much time it has gotten very, very little attention, this fixes that and completely changes the gateway destination to have much more than it currently does both in terms of area to explore, story to learn, and stuff to grab.

## Why It's Good For The Game

As of right now outpost45 sat at a spot where people would wait to see what the gateway destination is, see its outpost45 by the ominous purple glow, and then immediately back away knowing that outpost45, in a casual conversation in a ticket I was talking with an admin and they said "Well then if you don't like it anybody can contribute" and so I did. (Extreme thank you to pumpkin0, vekkter, and jolly for helping me learn how to PR, how to github, and how to map.)

## Changelog

Adds two new areas complete with their own threats and rewards,
Adds an old crowbar to the engineering storage
Generally makes the gateway have a much more engaging story with actual notes. (All be it shortly written because I wrote it at 2 am)

:cl:
add: More stuff to outpost 45 including two new areas full of hostiles
/:cl:

### CODE REVIEW

**Comment by Jolly-66** [comment]
You also have active turfs.

image

The tiles used on the shuttle have a different initial gas mix than the dirt/sand. 
image

**Review by Fikou — CHANGES_REQUESTED** [review]
The way you did the papers is really bad, you should code them like other away missions do, (see https://github.com/tgstation/tgstation/blob/master/code/modules/awaymissions/mission_code/caves.dm) and place that in this file https://github.com/tgstation/tgstation/blob/master/code/modules/awaymissions/mission_code/undergroundoutpost45.dm

**Comment by Yobrocharlie** [comment]
> You also have active turfs.
> 
> image
> 
> The tiles used on the shuttle have a different initial gas mix than the dirt/sand. image

ugh,
let me see if there is something i can do about this because i dont even know where to start.

**Comment by Yobrocharlie** [comment]
> The way you did the papers is really bad, you should code them like other away missions do, (see https://github.com/tgstation/tgstation/blob/master/code/modules/awaymissions/mission_code/caves.dm) and place that in this file https://github.com/tgstation/tgstation/blob/master/code/modules/awaymissions/mission_code/undergroundoutpost45.dm

I just did it through strongDMM because i dont actually know how to code

File: _maps/RandomZLevels/undergroundoutpost45.dmm [hunk 1]
```diff
@@ -10588,13 +11692,158 @@
 	heat_capacity = 1e+006
 	},
 /area/awaymission/undergroundoutpost45/research)
+"Gz" = (
+/obj/item/paper{
+	default_raw_text = Heythere,theminingoperationsaregoingswell,howeveriswearifeelsmallvibrationseverynowandthen.
```
Comment (ZephyrTFA) [inline _maps/RandomZLevels/undergroundoutpost45.dmm:11697]: This is why MDB died.

**Comment by ExcessiveUseOfCobblestone** [comment]
god I hate gateways

**Comment by Yobrocharlie** [comment]
I think I fixed it

**Comment by Yobrocharlie** [comment]
> The way you did the papers is really bad, you should code them like other away missions do, (see https://github.com/tgstation/tgstation/blob/master/code/modules/awaymissions/mission_code/caves.dm) and place that in this file https://github.com/tgstation/tgstation/blob/master/code/modules/awaymissions/mission_code/undergroundoutpost45.dm

I just removed all of the notes and fixed the atmos of the ship to be the same as the cave.

**Comment by Yobrocharlie** [comment]
i dont think i fixed everything

**Comment by Yobrocharlie** [comment]
I DID IT, ALL BUGS ARE PATCHED, RAH.

**Comment by Yobrocharlie** [comment]
Removed a few things and replaced them with a shovel

**Comment by Singul0** [comment]
non-diagonals for the mining pod walls for the "sec" area would make it less ugly

**Comment by Yobrocharlie** [comment]
done

**Comment by Singul0** [comment]
Go yell at a maptainer to review your PR or something man

**Comment by Maurukas** [comment]
Please resolve your conflicts.

**Comment by Yobrocharlie** [comment]
I won’t be able to resolve the conflicts for at least an hour or so, I’m getting a haircut, however can, in the mean time can somebody look over it to determine if this is even getting merged or not

**Review by Maurukas — CHANGES_REQUESTED** [review]
These ladders do not appear to be working:
image

There are heat capacity varedits. everywhere on this map. These were cleaned up in #75765. I'm not aware that there is any legitimate reason to edit this variable in a map. These crept in and were likely copy-pasted around. So these need to be removed before it can be merged:
image

There are temperature varedits on turfs with initial gas mix edits. The temperature is different than that in the gas mix and this seems to be triggering a large amount of active turfs and active groups on load. I do not believe there is a reason to edit this variable when an initial gas mix is provided, so these can just be removed:
image

**Comment by Yobrocharlie** [comment]
All of the turfs seem to be edited in outpost 45 due to its unique nature of that being of plasma instead of a normal gas mixture, i attempted to make the gas mix equal throughout all the tiles including the new special ones i included,

As for the ladders not working-
i have no idea how to make them work through strongDMM

**Comment by Singul0** [comment]
> All of the turfs seem to be edited in outpost 45 due to its unique nature of that being of plasma instead of a normal gas mixture, i attempted to make the gas mix equal throughout all the tiles including the new special ones i included,
> 
> As for the ladders not working- i have no idea how to make them work through strongDMM

make a new subtype for the tiles you use in the map, change it's atmos_mix. change the ladders ID? or just use a lift for it

**Comment by Yobrocharlie** [comment]
THE AMOUNT OF FUCKING CONFLICTS HAS GONE FROM 9 TO 11 TO FUCKING 49 WHAT THE FUCK IS THIS

**Comment by Yobrocharlie** [comment]
I TRIED TO REVERT MY CHANGES AND NOW ITS 172
WHY, FUCKING, WHY!!!

**Comment by Yobrocharlie** [comment]
AND NOW THE MAP WONT EVEN WORK PROPERLY IN STRONGDMM

**Comment by Yobrocharlie** [comment]
fuck it we try this again.

**Pull request closed without merging**